workloads/devstack-xenial.yaml

```diff
@@ -0,0 +1,55 @@
+---
+inherits: xenial
+needs_nested_vm: true
+vm:
+  mem_mib: 6144
+  cpus: 4
+  ports:
+  - host: 55580
+    guest: 80
+...
+---
+{{- define "ENV" -}}
+{{proxyVars .}}
+{{- print " DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true " -}}
+{{end}}
+{{ define "DEVSTACKPATH" }}{{ if $.MountPath "devstack" }}{{$.MountPath "devstack"}}{{else}}/home/{{.User}}/devstack{{end}}{{end}}
+#cloud-config
+write_files:
+ - content: |
+     [url "http://"]
+       insteadOf=git://
+   path: /home/{{.User}}/.gitconfig
```

Comment:
I'm not sure it's a good idea to use write_files to create files in the user'd directory.  The reason is that write_files runs before users-groups and so the user's directory will exist before useradd is called.  It might be better to do this in run_cmd.

Reply:
Agreed. Will fix it .